feat: preloads and nodeIntegration in iframes (#16425)
electron/electron: docs/api/web-contents.md

</html>
```

#### `contents.sendToFrame(frameId, channel[, arg1][, arg2][, ...])`

* `frameId` Integer
* `channel` String
* `...args` any[]

Send an asynchronous message to a specific frame in a renderer process via

> `channel`, you can also send arbitrary arguments. Arguments will be serialized

Reviewer: ```suggestion
`channel`. Arguments will be serialized
```

> in JSON internally and hence no functions or prototype chain will be included.

Reviewer: ```suggestion
as JSON internally and as such no functions or prototype chains will be included.
```
